doc/en/user/source/extensions/wps-longitudinal-profile/index.rst

```diff
@@ -0,0 +1,70 @@
+.. _wpslongitudinal:
+
+WPS longitudinal profile process
+================================
+
+WPS longitudinal profile process provides ability to calculate altitude profile for linestring
+* Reproject result to different CRS
+* Adjust altitude profile based on additional layer
+
+Installing the WPS longitudinal profile process
+-----------------------------------------------
+
+The WPS longitudinal profile extension is listed among the other extension downloads on the GeoServer download page.
+
+The installation process is similar to other GeoServer extensions:
+
+#. From the :website:`website download <download>` page, locate your release, and download: :download_extension:`wps-longitudinal-profile`
+   Verify that the version number in the filename corresponds to the version of GeoServer you are running (for example |release| above).
+
+#. Extract the contents of the archive into the :file:`WEB-INF/lib` directory in GeoServer.
+   Make sure you do not create any sub-directories during the extraction process.
+
+#. Restart GeoServer.
+
+
+Module description
+------------------
+
+This module provides longitudinal profile process.
+Process accepts following parameters:
+Required:
+1. layerName - name of the raster layer (coverage) which will be used, for altitude profile creation
+2. linestringEwkt - linestring in ewkt format, along which the altitude profile will be created
+3. distance - maximal distance between points of altitude profile
+Optional:
+1. adjustmentLayerName - name of the layer with altitude, which will be used to adjust altitude values
```

Comment:
A description of the expected data structure in the adjustment layer is required, for users to reproduce.
From what I'm seeing, it needs a polygon/multipolygon geometry, and a floating point attribute with an altitude delta, and the layer must be configured in GeoServer (I'm guessing it could be non advertised, if one does not like to have it publicly available.).

Reply:
fixed